ai/cryptocom-ai-agent-service/package.json.development

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Should we append the word .development to the package.json? Usually devDependencies contain the development dependencies and dependencies anything used in prod.

Reply:
removed